src/constants/index.js

```diff
@@ -1,3 +1,11 @@
+export const fontScalePresets = {
+  major_second: [1.802, 1.602, 1.424, 1.266, 1.125, 1],
+  minor_third: [2.488, 2.074, 1.728, 1.44, 1.2, 1],
+  major_third: [3.052, 2.441, 1.953, 1.563, 1.25, 1],
+  perfect_fourth: [4.209, 3.157, 2.369, 1.777, 1.333, 1],
+  augmented_fourth: [5.653, 3.998, 2.827, 1.999, 1.414, 1]
```

Comment:
Could you update these to be camelCased? I prefer that in JavaScript. Might even add a ESLint rule for it.

Reply:
Ya great call, I was blinding naming as the modularscale pattern was but you are right camelCase for the win